skimage/transform/_warps.py

```diff
@@ -665,6 +643,10 @@ def warp(image, inverse_map, map_args={}, output_shape=None, order=1,
     preserve_range : bool, optional
         Whether to keep the original range of values. Otherwise, the input
         image is converted according to the conventions of `img_as_float`.
+    resize : bool, optional
```

Comment:
I wonder if `resize` (the keyword we chose before) is the correct one: it does not include some notion of the translation needed to make the transform fit.  Maybe `fit_output` or `extend_to_fit`?